src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -97,7 +97,7 @@ public static function disableStty()
     /**
      * Asks the question to the user.
      *
-     * @return bool|mixed|string|null
+     * @return mixed
```

Comment:
`mixed` includes everything in PHP 8. Using `mixed|somethingElse` as a return type is a fatal error.
Obviously, we are going to handle the case in the `DebugClassLoader` when a PHPDoc is "wrong".
But in our own codebase, we should be strict and clean about it, WDYT?

Reply:
I agree that we should be strict and clean. But could be be more specific? 

`mixed` is the same as `string|int|float|bool|null|array|object|callable|resource`. 

I know that `Question::$default` has `mixed`, but that is also wrong. 

Is it reasonable to do anything else than `bool|string|null|int|float`?

Reply:
I did not look at the code here. `mixed` is probably wrong indeed, that's another fix to do.